tests/benchmarks/micro_benchmarks/test_nccl_bw_performance.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Tests for nccl-bw benchmark."""
+
+import numbers
+
+from tests.helper import decorator
+from superbench.benchmarks import BenchmarkRegistry, BenchmarkType, ReturnCode, Platform
+
+
+@decorator.cuda_test
+def test_kernel_launch_overhead():
+    """Test nccl-bw benchmark."""
+    context = BenchmarkRegistry.create_benchmark_context('nccl-bw', platform=Platform.CUDA, parameters='--gpu_count 1')
```

Comment:
gpu_count 1?

Reply:
If i remember correctly， test environment only have 1 gpu?

Reply:
For UT, it's not designed for the test machine actually.

Reply:
changed to mock UT according to Guoshuai's comments